happypose_ros/happypose_ros/inference_pipeline.py

```diff
@@ -99,7 +99,7 @@ def __call__(self, observation: ObservationTensor) -> Union[None, dict]:
 
         detections = self._wrapper.pose_predictor.detector_model.get_detections(
             observation,
-            output_masks=False,
+            output_masks=True,
```

Comment:
Maybe this should be parametrizable?

Reply:
The masks are produced in any case well calling the happypose detector so I don't think this adds much overhead. I'll use the `publish_seg_masks` parameter I'll add to activate it.

Reply:
done